templates/vue-starter-template/package.json

```diff
@@ -29,7 +29,8 @@
     "@shopware/helpers": "canary",
     "@shopware/nuxt-module": "canary",
     "nuxt": "3.17.5",
-    "vue": "3.5.16"
+    "vue": "3.5.16",
+    "@vuelidate/core": "2.0.3"
```

Comment:
this should go higher to match alphabetical order

Reply:
done